content/ai/concepts/ai-models/models/ai-diffusion-models/ai-diffusion-models.md

```diff
@@ -0,0 +1,42 @@
+---
+Title: 'Diffusion Models'
+Description:'Diffusion Models are generative models, which means they are used to generate data similar to what they were trained on. The models work by destroying training data through the addition of Gaussian noise, and then learning to recover that data.' 
+Subjects:
+  - 'Machine Learning'
+  - 'Data Science'
+  - 'Large Language Model'
+Tags:
+  - 'AI'
+  - 'Image Generation Model' 
+  - 'Denoising'
+  - 'Prompts'
+  - 'Prompt Engineering'
+  - 'Text-To-Image'
+  - 'Deep Learning'
+CatalogContent:
+  - 'machine-learning'
+  - 'paths/data-science'
+---
+
+**AI Diffusion Models** Are generative models, which means they are used to generate data similar to what they were trained on. The models work by destroying training data through the addition of Gaussian noise, and then learning to recover that data.
+
+## More About Diffusion Models
```

Comment:
```suggestion
## About Diffusion Models

```
Blank line after header

Reply:
done

Reply:
forgot to add it, can I add it in? @SSwiniarski